README.md

```diff
@@ -18,30 +18,30 @@ This VS Code extension provides support for creating and editing XML documents,
 
   * Syntax error reporting
   * General code completion
-  * [Auto-close tags](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Features/XMLFeatures.md#xml-tag-auto-close)
+  * [Auto-close tags](https://github.com/redhat-developer/vscode-xml/blob/main/docs/Features/XMLFeatures.md#xml-tag-auto-close)
   * Automatic node indentation
   * Symbol highlighting
   * Document folding
   * Document links
-  * [Document symbols and outline](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Symbols.md)
-  * [Renaming support](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Features/XMLFeatures.md#rename-tag)
-  * *[Automatic Tag Renaming](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Features/XMLFeatures.md#auto-rename-tag) when `editor.linkedEditing` is enabled*
-  * [Document Formatting](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md)
-  * [DTD validation](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Validation.md#validation-with-dtd-grammar)
+  * [Document symbols and outline](https://github.com/redhat-developer/vscode-xml/blob/main/docs/Symbols.md)
+  * [Renaming support](https://github.com/redhat-developer/vscode-xml/blob/main/docs/Features/XMLFeatures.md#rename-tag)
+  * *[Automatic Tag Renaming](https://github.com/redhat-developer/vscode-xml/blob/main/docs/Features/XMLFeatures.md#auto-rename-tag) when `editor.linkedEditing` is enabled*
+  * [Document Formatting](https://github.com/redhat-developer/vscode-xml/blob/main/docs/Formatting.md)
+  * [DTD validation](https://github.com/redhat-developer/vscode-xml/blob/main/docs/Validation.md#validation-with-dtd-grammar)
   * DTD completion
   * DTD formatting
-  * [XSD validation](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Validation.md#validation-with-xsd-grammar)
+  * [XSD validation](https://github.com/redhat-developer/vscode-xml/blob/main/docs/Validation.md#validation-with-xsd-grammar)
   * XSD based hover
   * XSD based code completion
   * XSL support
-  * [XML catalogs](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Validation.md#xml-catalog-with-xsd)
+  * [XML catalogs](https://github.com/redhat-developer/vscode-xml/blob/main/docs/Validation.md#xml-catalog-with-xsd)
   * File associations
   * Code actions
   * Schema Caching
 
 See the [changelog](CHANGELOG.md) for the latest release.
 
-You might also find useful information in the [Online XML Documentation](https://github.com/redhat-developer/vscode-xml/blob/master/docs/README.md)
+You might also find useful information in the [Online XML Documentation](https://github.com/redhat-developer/vscode-xml/blob/main/docs/README.md)
 or you can read this documentation inside vscode with the command `Open XML Document` available with `Ctrl+Shift+P`:
 
 ![XML Open Documentation](https://raw.githubusercontent.com/redhat-developer/vscode-xml/master/docs/images/Commands/XMLCommands.png)
```

Comment:
XML Open Documentation also needs to be changed

Reply:
Good catch. I was probably replacing based on `vscode-xml/blob/master/` or `vscode-xml/tree/master` so I missed this.